src/decoratedSensor.ts

```diff
@@ -30,16 +34,18 @@ export default class DecoratedSensor {
       return "";
     }
     const data = await this.getData();
+
     if(data.AQI === undefined) { return "" }
 
     console.log(this.currentAQINotifyBracket, this.formatReport(data));
-    const newBracket = this.calculateAQINotifyBracket(data.AQI) || this.currentAQINotifyBracket;
-    this.calculateAQIThresholds(data.AQI);
-    if(newBracket && newBracket !== this.currentAQINotifyBracket) {
-      this.currentAQINotifyBracket = this.calculateAQINotifyBracket(data.AQI);
```

Comment:
This is the "magic" line that becomes way more explicit in the updateAQIMonitoringState method